ClientTests/MockProfile.swift

```diff
@@ -88,6 +88,10 @@ public class MockProfile: Profile {
     }()
 
     lazy var bookmarks: protocol<BookmarksModelFactory, ShareToDestination> = {
+        // Make sure the rest of our tables are initialized before we try to read them!
+        // This expression is for side-effects only.
+        let p = self.places
```

Comment:
It's synchronous. The reason we do it here is that `SQLiteBookmarks` doesn't depend on `places`, but `places` is what pokes `BrowserTable` and makes the `bookmarks` table appear!

So our two choices are: 1. introduce this side-effect dependency, or 2. init `BrowserTable` twice, which is a waste. We went for the former under analysis, and this does the same.